client-runtime/src/main/java/com/microsoft/rest/v2/RestProxy.java

```diff
@@ -32,6 +32,7 @@
 import rx.Single;
 import rx.exceptions.Exceptions;
 import rx.functions.Func1;
+import sun.plugin.dom.exception.InvalidStateException;
```

Comment:
Checkstyle might be mad about this?

Reply:
I always forget to run checkstyle after I make little changes to a pull request. I'll fix this.